x-pack/plugin/eql/src/main/java/org/elasticsearch/xpack/eql/expression/function/scalar/string/CIDRMatch.java

```diff
@@ -105,7 +106,7 @@ public ScalarFunction makeSubstitute() {
         ScalarFunction func = null;
 
         for (Expression address : addresses) {
-            final Equals eq = new Equals(source(), field, address);
+            final Equals eq = new Equals(source(), field, address, UTC);
```

Comment:
For Equals without timezone I think it's best to create a constructor that doesn't require the tz , have the tz as null and potentially validate that it's not being used.
The tz should not be used in this case so forcing its user to specify one is just going to lead to problems.

Reply:
Could you explain a bit more about this validation that you mention? Validation for the CIDRMatch function or for the Equals?